Content.Server/Medical/CryoPodSystem.cs

```diff
@@ -163,10 +164,14 @@ private void OnActivateUIAttempt(EntityUid uid, CryoPodComponent cryoPodComponen
 
     private void OnActivateUI(EntityUid uid, CryoPodComponent cryoPodComponent, AfterActivatableUIOpenEvent args)
     {
+        TryComp<TemperatureComponent>(cryoPodComponent.BodyContainer.ContainedEntity, out var temp);
+        TryComp<BloodstreamComponent>(cryoPodComponent.BodyContainer.ContainedEntity, out var bloodstream);
+
         _userInterfaceSystem.TrySendUiMessage(
             uid,
             HealthAnalyzerUiKey.Key,
-            new HealthAnalyzerScannedUserMessage(cryoPodComponent.BodyContainer.ContainedEntity));
+            new HealthAnalyzerScannedUserMessage(cryoPodComponent.BodyContainer.ContainedEntity,
```

Comment:
The fact that this is the only medical data being passed in here shows you must be doing this the wrong way. I don't think you should be adding any code to this function.